.github/workflows/presubmit.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Presubmit
+
+on:
+
+  push:
+    branches: [ 'main', 'master' ]
+
+  pull_request:
+    branches: [ 'main', 'master', 'release-*' ]
```

Comment:
We don't actually need `master` and `release-*` right?

Reply:
If we need to patch release branches, we'll want that (I think).

All our other workflows work against `master`, and this has been useful for me. I keep my fork's default branch as `master` to easily (and lazily) differentiate it from this repo's `main` branch, so I don't accidentally push the wrong thing ever.